docker/camera/docker-compose.yaml

```diff
@@ -1,20 +1,16 @@
 version: '3.7'
 services:
   camera:
-    image: huntsman-pocs-camera:${TAG:-develop}
+    image: huntsmanarray/huntsman-pocs-camera:develop
     privileged: true
     network_mode: host
     stdin_open: true
     tty: true
     environment:
-      PANDIR:
-      POCS:
-      PANLOG:
-      HUNTSMAN_POCS:
-      PANOPTES_CONFIG_FILE:
+      PANDIR: /var/huntsman
       PANOPTES_CONFIG_HOST:
       PANOPTES_CONFIG_PORT:
     command: [ "huntsman-pyro --verbose service --service-class huntsman.pocs.camera.pyro.service.CameraService" ]
     volumes:
-      - "~/.ssh:/home/huntsman/.ssh"
-      - "/var/huntsman/images:/var/huntsman/images"
+      - "${PANDIR}/images:/var/huntsman/images"
+      - "${PANDIR}/logs:/var/huntsman/logs"
```

Comment:
Just a note that this is a bit confusing because the interpolation is actually happening on the host itself, not inside the running container.  So `$PANDIR` must exist on the host even though it is explicitly set on line 10 inside the container. So `$${PANDIR}/logs` here would make it use the env var inside the container, which is odd usage.

Reply:
Yes - this is intentional as we want those variables to be expanded on the host. Also with the cloud init setup we can guarantee that the PANDIR variable will be set. I explicitly set it to `/var/huntsman` inside the container because for some reason (I think it may be the `pocs` base image) it was getting set to `/var/panoptes`.